### DISCUSSION TITLE
point-stat usage on new use case

### DISCUSSION THREAD

**Discussion by YakelynRJ** [dtcenter/METplus#999] [Use Cases] [answered]
Hello, 
Thanks for helping with generating a NetCDF file to be used by point_stat. 
I am trying to pair saildrone data with the ECMWF forecast. The scatter plot below shows sensible (QS) and latent heat (QL) fluxes, with each marker indicating a different time, with a 5-min resolution. 

<img width="785" alt="Screen Shot 2021-07-12 at 3 36 52 PM" src="https://user-images.githubusercontent.com/32494099/125364312-1629cd00-e327-11eb-9f63-6f840e9ac88f.png">

The forecast is obtained from here: (https://apps.ecmwf.int/datasets/data/tigge/levtype=sfc/type=cf/)

Type of level: Surface
Type: Controle forecast
Month: June - October 2019
Origin and time: ECMWF, 00:00:00
Step: all
Parameter: Surface latent heat flux, surface sensible heat flux

This data has 6-hours lead times. 
I think point_stat is the best tool to pair obs with the forecast, but I am not sure how to modify the config file given that time/lat/lon are all changing. 

I modified the  PointStatConfig_tutorial_run1.txt (see attached), and I noticed that the configuration file asks for climatology files but I don't want that and I am running into errors (see attached screenshot). 

Thanks in advance for any help.

Here's the output from running Point-Stat:

<img width="993" alt="125342279-8c1f3b80-e309-11eb-8a45-40e16ac9567d" src="https://user-images.githubusercontent.com/21087144/125511817-01d0eb04-ed53-43ac-b076-a66f9d548d3e.png">

[PointStatConfig_tutorial_run1.txt](https://github.com/dtcenter/METplus/files/6804750/PointStatConfig_tutorial_run1.txt)
<img width="558" alt="Screen Shot 2021-07-08 at 10 39 31 AM" src="https://user-images.githubusercontent.com/32494099/125364121-c6e39c80-e326-11eb-999d-31e43e50e0ef.png">

**Comment by jprestop** [comment, marked as answer]
Hi @YakelynRJ.  Thank you for your question.  I believe that @JohnHalleyGotway is the best person to help you.  Please allow a few business days for a response.

**Reply by JohnHalleyGotway** [reply]
@YakelynRJ I see you're running Point-Stat to compare a GRIB2 forecast file (forecast_october_2019.grib2) to point observations. The "No matching records found" means that MET isn't able to interpret any of you records in that GRIB2 file as matching the "LHTFL" string. I'd recommend using wgrib2 to inventory that file:
```
wgrib2  forecast_october_2019.grib2
```
And check to see what string used for that variable. If it differs from LHTFL, please try switching the MET configuration to the new name. If you'd like, you could post that file you're using to our anonymous FTP site following the [How to send us data](https://github.com/dtcenter/METplus/discussions/954) instructions. I could go grab that file and let you know what configuration is needed.

FYI, when getting going with new datasets, I always start by running MET's plot_data_plane tool. That ensures that MET is able to read the data you expect, orient it correctly, and place it at the right spot on the earth. Please let me know if you post data there, and I can (hopefully) send you a command to have MET plot it.

**Reply by YakelynRJ** [reply]
Thanks, @JohnHalleyGotway. 

I posted the forecast data and the obs txt files in ramos_data using ftp. I originally generated nc files using ascii2nc and I used that an in input to point-stat. I will appreciate any help.

**Reply by JohnHalleyGotway** [reply]
@YakelynRJ I see that this discussion is related to the METplus Discussion #993. And I just replied to that discussion with more details about running plot_data_plane. I recommend that we Lock this discussion now to prevent future posts. But we can continue the discussion on #993. I did check the ftp site for a "ramos_data" directory but unfortunately it has aged off.

I'll wait for your confirmation before locking this discussion.

Thanks,
John
